src/App.jsx

```diff
@@ -1,14 +1,29 @@
 import React from 'react';
 import './App.css';
+import { Sum } from './components/Sum/Sum';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+      b={0}
```

Comment:
<img width="192" alt="Screenshot 2023-07-04 at 15 34 53" src="https://github.com/mate-academy/react_sum/assets/135232001/d908533d-01c1-4e3c-984b-43ba0b78e8cd">
Remove the 'b' parameter it's redundant here